mckhannah1901/Co-op-Blog-Project
# Ath 1126

### JIRA
[ATH-1126-Improve_Users_UI](https://wpengine.atlassian.net/secure/RapidBoard.jspa?rapidView=522&projectKey=ATH&modal=detail&selectedIssue=ATH-1126)

### What was done
- General UI improvements
- Incorporated some React components
- Tested these components with Jest

### How to test
- Run `yarn jest`
- Check the .tsx, .test.js and the relevant .html.erb files to see if React has been incorporated correctly
